src/Sylius/Component/Core/Model/ImageInterface.php

```diff
@@ -14,11 +14,22 @@
 use Sylius\Component\Resource\Model\ResourceInterface;
 use Sylius\Component\Resource\Model\TimestampableInterface;
 
-interface ImageInterface extends
-    ResourceInterface,
-    TimestampableInterface
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+interface ImageInterface extends ResourceInterface, TimestampableInterface
```

Comment:
Do we need to implement `TimestampableInterface`? Don't think it's useful for image...